pkg/generated/controllers/management.cattle.io/v3/feature.go

```diff
@@ -0,0 +1,376 @@
+	c.AddGenericHandler(ctx, name, FromFeatureHandlerToHandler(sync))
+}
+
+func (c *featureController) OnRemove(ctx context.Context, name string, sync FeatureHandler) {
+	c.AddGenericHandler(ctx, name, generic.NewRemoveHandler(name, c.Updater(), FromFeatureHandlerToHandler(sync)))
+}
+
+func (c *featureController) Enqueue(name string) {
+	c.controller.Enqueue("", name)
+}
+
+func (c *featureController) EnqueueAfter(name string, duration time.Duration) {
+	c.controller.EnqueueAfter("", name, duration)
+}
+
+func (c *featureController) Informer() cache.SharedIndexInformer {
+	return c.controller.Informer()
+}
+
+func (c *featureController) GroupVersionKind() schema.GroupVersionKind {
+	return c.gvk
+}
+
+func (c *featureController) Cache() FeatureCache {
+	return &featureCache{
+		indexer:  c.Informer().GetIndexer(),
+		resource: c.groupResource,
+	}
+}
+
+func (c *featureController) Create(obj *v3.Feature) (*v3.Feature, error) {
+	result := &v3.Feature{}
+	return result, c.client.Create(context.TODO(), "", obj, result, metav1.CreateOptions{})
+}
+
+func (c *featureController) Update(obj *v3.Feature) (*v3.Feature, error) {
+	result := &v3.Feature{}
+	return result, c.client.Update(context.TODO(), "", obj, result, metav1.UpdateOptions{})
+}
+
+func (c *featureController) UpdateStatus(obj *v3.Feature) (*v3.Feature, error) {
+	result := &v3.Feature{}
+	return result, c.client.UpdateStatus(context.TODO(), "", obj, result, metav1.UpdateOptions{})
+}
+
+func (c *featureController) Delete(name string, options *metav1.DeleteOptions) error {
+	if options == nil {
+		options = &metav1.DeleteOptions{}
+	}
+	return c.client.Delete(context.TODO(), "", name, *options)
+}
+
+func (c *featureController) Get(name string, options metav1.GetOptions) (*v3.Feature, error) {
+	result := &v3.Feature{}
+	return result, c.client.Get(context.TODO(), "", name, result, options)
+}
+
+func (c *featureController) List(opts metav1.ListOptions) (*v3.FeatureList, error) {
+	result := &v3.FeatureList{}
+	return result, c.client.List(context.TODO(), "", result, opts)
+}
+
+func (c *featureController) Watch(opts metav1.ListOptions) (watch.Interface, error) {
+	return c.client.Watch(context.TODO(), "", opts)
+}
+
+func (c *featureController) Patch(name string, pt types.PatchType, data []byte, subresources ...string) (*v3.Feature, error) {
+	result := &v3.Feature{}
+	return result, c.client.Patch(context.TODO(), "", name, pt, data, result, metav1.PatchOptions{}, subresources...)
+}
+
+type featureCache struct {
+	indexer  cache.Indexer
+	resource schema.GroupResource
+}
+
+func (c *featureCache) Get(name string) (*v3.Feature, error) {
+	obj, exists, err := c.indexer.GetByKey(name)
+	if err != nil {
+		return nil, err
+	}
+	if !exists {
+		return nil, errors.NewNotFound(c.resource, name)
+	}
+	return obj.(*v3.Feature), nil
+}
+
+func (c *featureCache) List(selector labels.Selector) (ret []*v3.Feature, err error) {
+
```

Comment:
nit: this is the only one with a space after the func declaration

Reply:
This file is generated, tried to undo and `go generate` just reverted. The other comparable files have the same at the same line.

Reply:
I did miss that it was in the generated directory - my bad!

Reply:
@thedadams ^